caikit/core/data_model/data_backends/dict_backend.py

```diff
@@ -63,7 +63,10 @@ def get_attribute(self, data_model_class: Type[DataBase], name: str) -> Any:
         #   being used correctly.
 
         # Make sure the name is a valid field on the given class
-        if name not in data_model_class.fields:
+        if (
```

Comment:
In general, anywhere that we assume a field needed to be in `cls.fields` it's now valid to check `cls._fields_oneofs_map` to allow the names of oneofs through.